### PR TITLE
Switch install.portworx.com:8080 to install.../1.3.0

### PR DESCRIPTION
**Pull request by zoxpx**
Switching to terraform-generated URLs

### CODE REVIEW

File: scheduler/kubernetes/upgrade-1.3.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ You can invoke the upgrade script with _--scaledownsharedapps off_ to skip scali
 
 For example:
 ```
-curl -fsL http://install.portworx.com:8080/upgrade | bash -s -- --scaledownsharedapps off
+curl -fsL http://install.portworx.com/1.3.0/upgrade | bash -s -- --scaledownsharedapps off
```
Comment (harsh-px) [inline scheduler/kubernetes/upgrade-1.3.md]: https

File: scheduler/kubernetes/upgrade-1.3.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ You can invoke the upgrade script with the _-t_ to override the default Portworx
 For example below command upgrades Portworx to _portworx/oci-monitor:1.3.0-rc5_ image.
 
 ```
-curl -fsL http://install.portworx.com:8080/upgrade | bash -s -- -t 1.3.0-rc5
+curl -fsL http://install.portworx.com/1.3.0/upgrade | bash -s -- -t 1.3.0-rc5
```
Comment (harsh-px) [inline scheduler/kubernetes/upgrade-1.3.md]: https

File: scheduler/kubernetes/upgrade-1.3.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ You are running Portworx as OCI if the Portworx daemonset image is _portworx/oci
 To upgrade, run the below curl command.
 
 ```
-curl -fsL http://install.portworx.com:8080/upgrade | bash -s 
+curl -fsL http://install.portworx.com/1.3.0/upgrade | bash -s
```
Comment (harsh-px) [inline scheduler/kubernetes/upgrade-1.3.md]: https

File: scheduler/kubernetes/upgrade-1.3.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ minion2 192.168.56.71   3.316327   4.1 GB     ...   1.2.11.10-421c67f   Online
 If the upgrade job crashes unexpectedly and fails to restore shared applications back to their original replica counts, you can run the following command to restore them.
 
 ```
-curl -fsL http://install.portworx.com:8080/upgrade | bash -s -- --scaledownsharedapps off
+curl -fsL http://install.portworx.com/1.3.0/upgrade | bash -s -- --scaledownsharedapps off
```
Comment (harsh-px) [inline scheduler/kubernetes/upgrade-1.3.md]: https

**Comment by zoxpx** [comment]
Thanks Harsh -- will wait for Travis, and then merge the changes

**Pull request merged**